release/release_tests.yaml

```diff
@@ -3741,7 +3741,7 @@
     script: pytest smoke_tests/smoke_test_basic_multi_node_training_learner.py
 
     wait_for_nodes:
-      num_nodes: 2
+      num_nodes: 3
```

Comment:
This makes is so that we wait for the head node plus the two worker nodes required for some of the tests.